pom.xml

```diff
@@ -10,26 +10,19 @@
     <packaging>jar</packaging>
 
     <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <camel.version>3.0.1</camel.version>
+        <camel.version>3.3.0</camel.version>
+        <spring-boot.version>2.3.0.RELEASE</spring-boot.version>
         <datasonnet.version>1.0.5-SNAPSHOT</datasonnet.version>
         <junit.version>4.12</junit.version>
         <junit.jupiter.version>5.5.2</junit.jupiter.version>
     </properties>
 
     <build>
         <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.8.1</version>
-                <configuration>
-                    <source>1.8</source>
```

Comment:
Moving on from Java 8 compatibility would be a pretty big change, we're going to need to discuss it on our end. I'm inclined to support it for this specific library, since it's for use with Camel and our Camel codebases are all newer and we control the runtime. This is unlike Datasonnet generally, which definitely needs to remain 1.8 compatible.

Reply:
Oh! Actually I just replaced the explicit dependency with these lines 
```
        <maven.compiler.source>1.8</maven.compiler.source>
        <maven.compiler.target>1.8</maven.compiler.target>
```
these are the properties that the compiler plugin looks for, so it's still java 8

Reply:
Ah, I totally missed it, perfect